src/Onebip/DateTime/UTCDateTime.php

```diff
@@ -423,6 +423,27 @@ public function startOfMonth()
         );
     }
 
+    public function endOfMonth()
+    {
+	return self::box(
+	    $this->toDateTime()->format('Y-m-t')
```

Comment:
This `format` is hidden by the name of the function, let a new name of the function reveal it.